managers/engineering_manager.md

```diff
@@ -27,28 +29,28 @@ Engineering Manager
 
 #### Meetings
 
-* Guide effectiveness of team meetings. Ensure agendas/purpose are clear & actions captured. 
+* Guide effectiveness of team meetings.
+* Ensure agendas/purpose are clear & actions captured.
 
 #### Determine team/team makeup & hiring needs
 
 * Ensure team has the people and skill sets needed to achieve their goals
-* Recruit for the team 
+* Recruit for the team
```

Comment:
@adsteel / @jcarbaugh / @cathrynsupko / @btjones - asking directors here, does this section seem accurate to how we have things structured. Asking for a friend lol _Am I doing my job correctly?_ 🆒 😢 

I sort of feel maybe recruitment here could use some sharpening in the language so as to reflect what is and isn't within scope/ability?

Reply:
Good point. Hiring is still part of everyone's job, I think, but recruiting isn't so much anymore.